### PR TITLE
removed store name and icon from app bar. hide top share button. move…

### PR DESCRIPTION
**Pull request by mdobosh-smarttech**
…ed search to le left. hide store info panel. hide bottom nav. bar.

**What does this PR do?**

   A few sentences describing the overall goals of the pull request's commits. 

   Explain any technical decisions that were made and why (when it makes sense).

**Database changed?**

   Yes | No

**Where should the reviewer start?**

- [ ] Foo.java
- [ ] Bar.kt

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [APP-XXXX](https://aptoide.atlassian.net/browse/APP-XXXX)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-XXXX](https://aptoide.atlassian.net/browse/APP-XXXX)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

### CODE REVIEW

File: app/src/main/java/cm/aptoide/pt/view/BaseFragment.java [hunk 1]
```diff
@@ -47,13 +47,6 @@ private static long getRemovingParentAnimationDuration(Fragment fragment, long d
     }
   }
 
-  @Override public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
```
Comment (mdobosh-smarttech) [inline app/src/main/java/cm/aptoide/pt/view/BaseFragment.java:50]: Hides bottom nav bar

File: app/src/main/java/cm/aptoide/pt/view/recycler/displayable/DisplayablesFactory.java [hunk 2]
```diff
@@ -133,11 +133,7 @@
           }
 
         case HOME_META:
-          return Observable.just(
```
Comment (mdobosh-smarttech) [inline app/src/main/java/cm/aptoide/pt/view/recycler/displayable/DisplayablesFactory.java:136]: Removing app store meta-data

File: app/src/main/java/cm/aptoide/pt/view/recycler/displayable/DisplayablesFactory.java [hunk 3]
```diff
@@ -156,10 +152,7 @@
                   windowManager, resources, storeTabNavigator, navigationTracker, themeManager));
 
         case COMMENTS_GROUP:
-          return Observable.from(
```
Comment (mdobosh-smarttech) [inline app/src/main/java/cm/aptoide/pt/view/recycler/displayable/DisplayablesFactory.java:159]: Removing comments/rate

File: app/src/main/java/cm/aptoide/pt/store/view/StoreFragment.java [hunk 4]
```diff
@@ -547,14 +555,11 @@ protected boolean displayHomeUpAsEnabled() {
...
   @Override protected void setupToolbarDetails(Toolbar toolbar) {
-    toolbar.setTitle(title);
-    toolbar.setBackgroundResource(
-        themeManager.getAttributeForTheme(storeTheme, R.attr.toolbarBackground).resourceId);
-    if (userId != null) {
-      toolbar.setLogo(R.drawable.ic_user_shape_white);
-    } else {
-      toolbar.setLogo(R.drawable.ic_store_white);
-    }
+    toolbar.setTitle(null);
```
Comment (mdobosh-smarttech) [inline app/src/main/java/cm/aptoide/pt/store/view/StoreFragment.java:558]: Creating search icon on the left side (hiding search MenuItem in right Menu)

File: app/src/main/java/cm/aptoide/pt/store/view/StoreFragment.java [hunk 5]
```diff
@@ -192,6 +195,10 @@ public static StoreFragment newInstance(String storeName, String storeTheme) {
 
       final MenuItem menuItem = menu.findItem(R.id.menu_item_search);
       if (appSearchSuggestionsView != null && menuItem != null) {
+        if (searchIcon != null) {
+          searchIcon.setOnClickListener((v) -> menu.performIdentifierAction(R.id.menu_item_search, 0));
```
Comment (mdobosh-smarttech) [inline app/src/main/java/cm/aptoide/pt/store/view/StoreFragment.java:199]: Pass click

**Review by osalamakhagl — APPROVED** [review]

**Pull request merged**